vendor/data/codelists/BEIS/programme_status_to_iati_status_mapping.yml

```diff
@@ -0,0 +1,23 @@
+data:
```

Comment:
How about defining this mapping in the `ProgrammeToIatiStatus` class itself, e.g.

```ruby
STATUS_MAPPING = {
  "01" => "2",
  "02" => "1",
  "03" => "1"
  ... etc...
}.freeze
```

Reply:
That makes more sense. Thanks leeky!